test/Nancy.Tests/project.json

```diff
@@ -1,6 +1,8 @@
 {
+    "version": "1.4.1.0",

```

Comment:
Without this, `CookieBasedSessionsFixture.Should_load_valid_test_data` fails because the `DefaultObjectSerializer` can't find Nancy.Tests, Version=1.4.1.0, Culture=neutral, PublicKeyToken=null, which was presumably encoded inside `CookieBasedSessionsFixture.ValidData`.
The full stack trace of the failure is

```
System.IO.FileLoadException : Could not load file or assembly 'Nancy.Tests, Version=1.4.1.0, Culture=neutral, PublicKeyToken=null'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
Stack Trace:
     at System.RuntimeTypeHandle.GetTypeByName(String name, Boolean throwOnError, Boolean ignoreCase, Boolean reflectionOnly, StackCrawlMarkHandle stackMark, IntPtr pPrivHostBinder, Boolean loadTypeFromPartialName, ObjectHandleOnStack type, ObjectHandleOnStack keepalive)
     at System.RuntimeTypeHandle.GetTypeByName(String name, Boolean throwOnError, Boolean ignoreCase, Boolean reflectionOnly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean loadTypeFromPartialName)
     at System.RuntimeType.GetType(String typeName, Boolean throwOnError, Boolean ignoreCase, Boolean reflectionOnly, StackCrawlMark& stackMark)
     at System.Type.GetType(String typeName)
     at CallSite.Target(Closure , CallSite , Type , Object )
  src\Nancy\DefaultObjectSerializer.cs(69,0): at Nancy.DefaultObjectSerializer.Deserialize(String sourceString)
  src\Nancy\Session\CookieBasedSessions.cs(190,0): at Nancy.Session.CookieBasedSessions.Load(Request request)
  test\nancy.tests\Unit\Sessions\CookieBasedSessionsFixture.cs(370,0): at Nancy.Tests.Unit.Sessions.CookieBasedSessionsFixture.Should_return_blank_session_if_hmac_changed()
System.IO.FileLoadException : Could not load file or assembly 'Nancy.Tests, Version=1.4.1.0, Culture=neutral, PublicKeyToken=null'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
Stack Trace:
     at System.RuntimeTypeHandle.GetTypeByName(String name, Boolean throwOnError, Boolean ignoreCase, Boolean reflectionOnly, StackCrawlMarkHandle stackMark, IntPtr pPrivHostBinder, Boolean loadTypeFromPartialName, ObjectHandleOnStack type, ObjectHandleOnStack keepalive)
     at System.RuntimeTypeHandle.GetTypeByName(String name, Boolean throwOnError, Boolean ignoreCase, Boolean reflectionOnly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean loadTypeFromPartialName)
     at System.RuntimeType.GetType(String typeName, Boolean throwOnError, Boolean ignoreCase, Boolean reflectionOnly, StackCrawlMark& stackMark)
     at System.Type.GetType(String typeName)
     at CallSite.Target(Closure , CallSite , Type , Object )
  src\Nancy\DefaultObjectSerializer.cs(69,0): at Nancy.DefaultObjectSerializer.Deserialize(String sourceString)
  src\Nancy\Session\CookieBasedSessions.cs(190,0): at Nancy.Session.CookieBasedSessions.Load(Request request)
  test\nancy.tests\Unit\Sessions\CookieBasedSessionsFixture.cs(355,0): at Nancy.Tests.Unit.Sessions.CookieBasedSessionsFixture.Should_load_valid_test_data()
```

This makes me a little concerned about the behaviour of `DefaultObjectSerializer`, especially if it's used to serialize and deserialize long-lived data, but I think that's outside the scope of this PR.